src/rules/no-named-default.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  meta: {
+    docs: {},
+  },
+
+  create: function (context) {
+    return {
+      'ImportDeclaration': function (node) {
+        if (node.source == null) return
```

Comment:
I'm curious, is there ever a case where the source is not defined? :question:
We can leave it in case you're not sure, but I don't when this would happen.